packages/wrangler/src/pages/index.ts

```diff
@@ -44,7 +45,7 @@ export function pages(yargs: CommonYargsArgv) {
 						Build.Handler
 					)
 					.command(
-						"build-env",
+						"build-env [projectDir]",
```

Comment:
I imagine we'd always run this from the Pages project directory. Is this needed for CI?

Reply:
This is needed for CI, yeah